.gitignore

```diff
@@ -16,6 +16,7 @@
 *.launch
 .settings/
 *.sublime-workspace
+.eslintrc
```

Comment:
not sure if this file should be ignored. since this defines the js coding style, and should be shared amongst all developers